src/Symfony/Installer/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony Installer package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Installer\Tests;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Filesystem\Filesystem;
+
+class IntegrationTest extends \PHPUnit_Framework_TestCase
+{
+    private static $rootDir = __DIR__.'/../../../../';
+    private $fs;
+
+    public function setUp()
+    {
+        $this->fs = new Filesystem();
+
+        $this->fs->remove(self::$rootDir.'/symfony.phar');
+        $this->runCommand('./vendor/bin/box build');
```

Comment:
I suggest building the phar only once using ``setUpBeforeClass`` instead of building it again for each test. It should make things faster